www/install/php/Update-20.04.0-beta.1.php

```diff
@@ -80,47 +78,79 @@
         }
         $statement->bindValue(':value', $fileName, \PDO::PARAM_STR);
         $statement->bindValue(':id', $row['bk_mod_id'], \PDO::PARAM_INT);
-        $statement->execute();
+        if (false === $statement->execute()) {
+            throw new \PDOException(
```

Comment:
The CentreonDB class is already configured to generate an exception if an error occurs.